docs/installation/windows.md

```diff
@@ -1,161 +1,263 @@
+- mingw-w64-x86_64-dlfcn
+- mingw-w64-x86_64-xerces-c
+- mingw-w64-x86_64-zeromq
+
+All packages can be installed by typing the following:
+```bash
+$ pacman -S --needed base-devel mingw-w64_x86_64-toolchain mingw-w64-x86_64-cmake mingw-w64-x86_64-boost mingw-w64-x86_64-qt5 mingw-w64-x86_64-dlfcn mingw-w64-x86_64-xerces-c
+```
+One issue that the boost package has when building with GridLAB-D with HELICS on MSYS2 is all the boost libraries have a -mt in their names. In order for GridLAB-D to find these libraries the -mt must be removed from the libboost*-mt.dll files located in /mingw64/bin. Work is in progress to remove this modification in MSYS2 builds.
+
+After all this has been done /mingw64/bin must be added to the PATH environment variable.
+```bash
+$ export PATH=$PATH:/mingw65/bin
+```
+
+### Download HELICS and GridLAB-D Source Code ###
+Now that the MSYS2 environment has been setup and all prerequisite packages have been installed the source code can be compiled and installed. The HELICS source code can be cloned from GitHub by performing the following:
+```bash
+$ git clone https://github.com/GMLC-TDC/HELICS-src.git
+```
+git will clone the source code into a folder in the current working directory called HELICS-src. This path will be refered to by this guide as HELICS_ROOT_DIR.
+
+The GridLAB-D source code can be cloned from GitHub by perfoming the following:
+```bash
+$ git clone https://github.com/gridlab-d/gridlab-d.git
+```
+git will clone the source code into a folder in the current working directory called gridlab-d. This path will be refered to by this guide as GLD_ROOT_DIR.
+
+As of the writting of this guide the only GridLAB-D branch that can be compiled with HELICS is the develop branch. So you must checkout the develop branch of the GridLAB-D repository. Change your working directory to GLD_ROOT_DIR and then checkout the develop branch:
+```bash
+$ git checkout -b develop origin/develop
+```
+
+### Compiling HELICS From Source ###
+Change directories to HELICS_ROOT_DIR. Create a directory called helics-build. This can be accomplished by using the mkdir command. cd into this directory. Now type the following:
+```bash
+$ cmake-gui ../
+```
+If this failes that is because mingw-w64-x86_64-qt5 was not installed. If you did install it the cmake gui window should pop up. click the Advanced check box next to the search bar. Then click Configure. A window will pop up asking you to specify the generator for this project. Select MSYS Makesfiles from the dropdown menu. Then click the Specify native compilers check box and click next. Enter C:/msys64/mingw64/bin/gcc.exe for the C compiler and C:/msys64/mingw64/bin/g++.exe for the C++ compiler and click finish. Once the Configure process finished several variables will show up highlighted in red. Since this is the first time setup the Boost and ZeroMQ library. Below are the following cmake variables that need to be verified.

```

Comment:
If we want this to be as easy as possible there should be a script provided that calls cmake directly with the appropriate variables.

Reply:
While it would make it easier on the developer I don't think the burden should be on us to provide a script to go out and find if someone installed the prerequisites in an nonstandard location.

Reply:
I think this comment was related to the changing of boost-mt files to not have that extension,  But I think that was resolved so the comment isn't relevant any more.